ob_intern/ShortVDOService.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  ShortVDOService.swift
+//  ob_intern
+//
+//  Created by Soft Liampisan on 29/6/2566 BE.
+//
+
+import Foundation
+import Alamofire
+import SwiftyJSON
+
+class ShortVDOService {
+    
+    func getShortVDOPost(completion: @escaping ( ([ShortVideoPost]) -> Void),
+                         errorHandler: @escaping ( (Error) -> Void)){
+        let request = AF.request("https://cccfefc5-f8d5-4f42-b9c0-1a17022434cb.mock.pstmn.io/getVideoPost",
```

Comment:
น่าจะอันง่ายกว่านิดนึงครับ
```suggestion
        let link = "https://cccfefc5-f8d5-4f42-b9c0-1a17022434cb.mock.pstmn.io/getVideoPost"
        let request = AF.request(link, method: .get)
```